server/matchmaker/matchmaker_queue.py

```diff
@@ -83,6 +85,9 @@ def get_map_pool_for_rating(self, rating: int) -> Optional[MapPool]:
                 continue
             return map_pool
 
+    def get_game_options(self) -> Dict[str, Any]:
+        return self.params.get("GameOptions") or None
```

Comment:
`get` already returns `None` when a key is missing

Reply:
Heh, yea but not if GameOptions is `{}` 😉